docs/user/index.rst

```diff
@@ -221,16 +222,16 @@ Keep your documentation up to date:
 Business features
 -----------------
 
-Features for organizations and businesses:
+Our Business hosting has everything your business needs:
 
 :doc:`/commercial/organizations`
-  Learn how to manage your organization on Read the Docs.
+  Control permissions across multiple teams.
```

Comment:
I'm not sure if it's better or not, but it's more natural to me the usage of _manage_ 🤷🏼 

```suggestion
  Manage permissions across multiple teams.
```